macros/models/_sector/transfers/eth/transfers_eth.sql

```diff
@@ -0,0 +1,71 @@
+{% macro transfers_eth(blockchain, eth_placeholder_contract, base_traces, base_transactions, erc20_transfer ) %}
+
+with eth_transfers as (
+    select
+        r."from"
+        ,r.to
+        --Using the ETH deposit placeholder address to match with prices tables
+        ,{{eth_placeholder_contract}} as contract_address
+        ,cast(r.value as double) AS value
+        ,cast(r.value as double)/1e18 as value_decimal
+        ,r.tx_hash
+        ,r.trace_address
+        ,r.block_time as tx_block_time
+        ,r.block_number as tx_block_number
+        ,substring(to_hex(t.data), 1, 10) as tx_method_id
+        ,cast(r.tx_hash as varchar) || '-' || array_join(r.trace_address,',') as unique_transfer_id
+        ,t.to AS tx_to
+        ,t."from" AS tx_from
+    from {{ base_traces }} as r
+    join {{ base_transactions}} as t
+        on r.tx_hash = t.hash
+        and r.block_number = t.block_number
+    where
+        (r.call_type not in ('delegatecall', 'callcode', 'staticcall') or r.call_type is null)
+        and r.tx_success
+        and r.success
+        and r.value > uint256 '0'
```

Comment:
Do we need to cast still? I believe there was some query engine update that made us not have to any more.

I think I remember some chains not using uint256